projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -32,6 +32,9 @@ export class MenuDropdownComponent {
   @Input()
   public label?: string;
 
+  @Input()
+  public disabled: boolean | undefined;
```

Comment:
Any styling changes to indicate that it's disabled?

Reply:
yep. got sidetracked by insights issues